Hermes/appscale/hermes/stats/stats_app.py

```diff
@@ -58,357 +53,155 @@ class HandlerInfo(object):
     """
-    return [
-      (route, handler.handler_class, handler.init_kwargs)
-      for route, handler in self._routes.iteritems()
-    ]
+    newer_than = time.mktime(datetime.utcnow().timetuple())
+    future_stats = stats_source.get_current_async(newer_than=newer_than)
+    IOLoop.current().add_future(future_stats, write_stats_callback)
 
-  def _init_local_node_stats_publisher(self):
-    """ Starts node stats publisher, creates cache for local node stats
-        and initializes corresponding API endpoints.
-    """
-    stats = self._local_node_stats
-    # Init cache
-    stats.cache = StatsCache(stats.cache_size)
-    # Init source
-    stats_source = NodeStatsSource()
-    # Configure stats publishing
-    stats.publisher = StatsPublisher(stats_source, stats.update_interval)
-    stats.publisher.subscribe(stats.cache)
-    self._publishers.append(stats.publisher)
-    # Configure handlers
-    self._routes['/stats/local/node/cache'] = HandlerInfo(
-      handler_class=CachedStatsHandler,
-      init_kwargs=dict(stats_cache=stats.cache)
-    )
-    self._routes['/stats/local/node/current'] = HandlerInfo(
-      handler_class=CurrentStatsHandler,
-      init_kwargs=dict(stats_source=stats_source)
-    )
+  PeriodicCallback(profiling_periodical_callback, interval).start()
 
-  def _init_local_processes_stats_publisher(self):
-    """ Starts processes stats publisher, creates cache for local proc. stats
-        and initializes corresponding API endpoints.
-    """
-    stats = self._local_processes_stats
-    # Init cache
-    stats.cache = StatsCache(stats.cache_size)
-    # Init source
-    stats_source = ProcessesStatsSource()
-    # Configure stats publishing
-    stats.publisher = StatsPublisher(stats_source, stats.update_interval)
-    stats.publisher.subscribe(stats.cache)
-    self._publishers.append(stats.publisher)
-    # Configure handlers
-    self._routes['/stats/local/processes/cache'] = HandlerInfo(
-      handler_class=CachedStatsHandler,
-      init_kwargs=dict(stats_cache=stats.cache)
-    )
-    self._routes['/stats/local/processes/current'] = HandlerInfo(
-      handler_class=CurrentStatsHandler,
-      init_kwargs=dict(stats_source=stats_source)
-    )
 
-  def _stub_processes_stats_routes(self):
-    """ Sets stub handlers to processes stats API endpoints
-    """
-    self._routes['/stats/local/processes/cache'] = HandlerInfo(
-      handler_class=Respond404Handler,
-      init_kwargs=dict(reason='Processes stats is disabled')
-    )
-    self._routes['/stats/local/processes/current'] = HandlerInfo(
-      handler_class=Respond404Handler,
-      init_kwargs=dict(reason='Processes stats is disabled')
-    )
-    self._routes['/stats/cluster/processes'] = HandlerInfo(
-      handler_class=Respond404Handler,
-      init_kwargs=dict(reason='Processes stats is disabled')
-    )
+def configure_node_stats_profiling():
+  """ Configure and start periodical callback for collecting and than
```

Comment:
Change to "Configures and starts periodical callbacks" here and few below doc strings.